.github/workflows/regression-tests.yml

```
@@ -168,6 +168,15 @@ jobs:
             cc: clang
             extra_name: "clang | "
 
+          - sim: icarus             # with gcov/gcovr
+            sim-version: v10_3
+            lang: verilog
+            python-version: 3.8
+            os: ubuntu-latest
+            extra_name: "gcovr | "
+            cxx: \"ccache g++\"
+            cc: \"ccache gcc\"
```

Some actual testing shows this should do the right thing.
```suggestion
            cxx: '"ccache g++"'
            cc: '"ccache gcc"'
```